.github/workflows/_reusable_surge-deploy-preview.yml

```diff
@@ -13,12 +13,14 @@ jobs:
   # IMPORTANT: the logic is duplicated in the `surge-deploy-pr-preview-test.yml` workflow. Keep both definitions in sync.
   deploy:
     runs-on: ubuntu-22.04
+    if: ${{ github.event.workflow_run.event == 'pull_request' && github.event.workflow_run.conclusion == 'success' }}
```

Comment:
**thought:** we already did that in the workflow we used to validate the solution but miss it here (probably a mistake)
See https://github.com/process-analytics/github-actions-playground/blob/56add9dacc916c898c609fbe480bd7ceb2ba353c/.github/workflows/surge-pr-fork-02-deploy.yml#L15-L17